x-pack/plugins/security_solution/cypress/e2e/detection_rules/prebuilt_rules.cy.ts

```diff
@@ -79,31 +64,27 @@ describe('Prebuilt rules', () => {
 
         cy.get(SELECT_ALL_RULES_ON_PAGE_CHECKBOX).click();
         enableSelectedRules();
-        waitForRuleToChangeStatus();
+        waitForRuleToUpdate();
```

Comment:
Offtopic, but... I'm curious how this test called `Allows to enable/disable all rules at once` ended up being a part of the `Rule monitoring table` set which is in turn a part of the test suite for prebuilt rules. How all that is related?

Reply:
🤷‍♂️ I don't think there's a good reason for that. This code is probably overdue for refactoring.